README.md

```diff
@@ -38,6 +57,24 @@ Notes:\\
 $I:=D^1$ is the interval used for the definition of a homotopy.
 
 \subsubsection*{Disjoint union}
+The Disjoint Union is most properly described in terms of universal properties,
+but because of that it has many equivalent concrete descriptions.  So we pick
+a totally arbitrary definition, the Disjoint Union of a collection of sets,
+$X_i$, indexed by $i\in I$, is just $\bigcup_{i\in I} \bigcup_{x\in X_i} (i, x)$.
+This is usually written $\bigsqcup_{i\in I} X_i$, or you could write it as a
+binary operation, $A \sqcup B$.\\
+Taking a concrete example the disjoint union of $\{a, b\}$ and $\{b, c\}$ is
+$\{(0,a), (0,b), (1,b), (1,c)\}$ if the sets are indexed by $0,1$.
+This is just a way of forcing sets to be disjoint before we take their unions,
+the reason for this is that if an element is in more than one of the sets,
```

Comment:
I think this should be 2 sentences

Reply:
Yeah probably, I do run on sentences a lot.